### PR TITLE
Fix for Ambari credentials

### PR DESCRIPTION
**Pull request by ivanayov**

### CODE REVIEW

**Comment by ivanayov** [comment]
@duncangrant , @bostko could you please review?

**Comment by bostko** [comment]
@iyovcheva Looks good but you missed to refactor `serviceUpHttpFeed`

**Comment by ivanayov** [comment]
@bostko `serviceUpHttpFeed` doesn't need credentials and we wouldn't like to build it twice as there is nothing to update

File: ambari/src/main/java/io/brooklyn/ambari/server/AmbariServerImpl.java [hunk 1]
```diff
@@ -137,30 +136,7 @@ protected void connectSensors() {
...
-                        ambariUri,
-                        "Cluster1",
-                        1))
-                .credentials(usernamePasswordCredentials.getUserName(), usernamePasswordCredentials.getPassword())
-                .header(HttpHeaders.AUTHORIZATION, HttpTool.toBasicAuthorizationValue(usernamePasswordCredentials))
-                .poll(new HttpPollConfig<String>(CLUSTER_STATE)
-                        .onSuccess(Functionals.chain(HttpValueFunctions.jsonContents(), getRequestState()))
-                        .onFailureOrException(Functions.<String>constant(null)))
-                .build();
+        connectSensorsHelper();
```
Comment (duncangrant) [inline ambari/src/main/java/io/brooklyn/ambari/server/AmbariServerImpl.java]: Is there a better way to name this - possibly connectAuthenticatedSensors? - just to try to make it obvious why it is separate from the rest.

**Comment by duncangrant** [comment]
I think the naming of the connectSensorsHelper/disconnectSensorsHelper needs fixed but otherwise this is fine and I'd be happy for this to be merged once these have been renamed.

**Pull request merged**